scripts/core/common.yml

```diff
@@ -407,6 +407,9 @@ etors:
     - name: DEVICE_MEMORY_EXT_PROPERTIES
       desc: $x_device_memory_ext_properties_t
       version: "1.4"
+    - name: DEVICE_IP_VERSION_EXT
```

Comment:
should the name be EXT_PROPERTIES as with others?

Reply:
no, we are ok.

Reply:
I considered this, but this query isn't returning any "properties", so I didn't think this made sense.

Note that the EU count query a few lines above also does not include "properties" in the name.

Reply:
right, I noticed the EU count as well. cool.